.github/dependabot.yml

```diff
@@ -1,4 +1,9 @@
 version: 2
+registries:
+  python-shopify:
+    type: python-index
+    url: https://pkgs.shopify.io/basic/data/python/simple
+    token: ${{secrets.PYTHON_CLOUDSMITH_TOKEN}}
```

Comment:
this looks good! you  also need to add `registries` key under `updates` like here https://github.com/Shopify/shopify/blob/master/.github/dependabot.yml#L27 so it's actually being used for that package-ecosystem

Reply:
oh, so sorry! it was the last line and i totally missed it 🙈